apps/playgrounds/arkenv-cli/src/index.css

```
@@ -0,0 +1,109 @@
+	--text: #6b6375;
+	--text-h: #08060d;
+	--bg: #fff;
+	--border: #e5e4e7;
+	--code-bg: #f4f3ec;
+	--accent: #aa3bff;
+	--accent-bg: rgba(170, 59, 255, 0.1);
+	--accent-border: rgba(170, 59, 255, 0.5);
+	--social-bg: rgba(244, 243, 236, 0.5);
+	--shadow: rgba(0, 0, 0, 0.1) 0 10px 15px -3px, rgba(0, 0, 0, 0.05) 0 4px 6px -2px;
+
+	--sans: system-ui, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.10.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)